tests/tips/test_tips_disputed_submissions.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+from brownie import accounts
+from brownie import Autopay
+from brownie import chain
+from brownie import TellorPlayground
+from telliot_core.tellor.tellor360.autopay import Tellor360AutopayContract
+
+from telliot_feeds.feeds import matic_usd_median_feed
+from telliot_feeds.reporters.tips.suggest_datafeed import get_feed_and_tip
+
+
+# deploy new contracts
+@pytest.fixture(scope="function")
+def deploy(query_data_storage_contract):
+    # deploy playground
+    playground = accounts[0].deploy(TellorPlayground)
+    # deploy autopay
+    autopay = accounts[0].deploy(Autopay, playground.address, query_data_storage_contract.address, 20)
+    return playground, autopay
+
+
+@pytest.mark.asyncio
+async def test_tips_disputed_submissions(deploy, mumbai_test_cfg):
```

Comment:
This is a great test addition, but I think you could make what's happening clearer for future readers/maintainers/contributors. The comments within the test function are helpful. Perhaps add a docstring explaining the overall flow and purpose of the test, similar to your explanation in the PR description.